addons/portal_rating/static/src/scss/portal_rating.scss

```diff
@@ -39,10 +36,39 @@ $o-w-rating-star-color: #FACC2E;
             font-size: $font-size-sm;
         }
         .o_website_rating_table_reset {
-            .o_website_rating_select_text {
-                visibility: hidden;
+            .o_website_rating_selection_reset {
+                color: $red;
+            }
+        }
+        // hide/show button to clear the filter
+        .o_website_rating_select.o_website_rating_table_row_selected {
+            .o_website_rating_table_reset {
+                .o_website_rating_selection_reset {
+                    visibility: visible;
+                }
             }
         }
```

Comment:
This comment is about the general structure of the ratings table, and would require changes in several files.
It is not exactly the scope of the task either, but as you do have to use and build on it, it may be worth it. 
Currently the layout looks like this:
```xml
<div class="o_website_rating_progress_table o_website_rating_progress_table_has_selection">
    <div class="o_website_rating_select">...</div>
    <div class="o_website_rating_select o_website_rating_table_row_selected">
        <div class="o_website_rating_table_star_num" />
        <div class="o_website_rating_table_progress" />
        <div class="o_website_rating_table_percent" />
        <div class="o_website_rating_table_reset" />
    <div class="o_website_rating_select">...</div>
    ...
    </div>
</div>
```
While we know that table rows are used to select a rating to filter messages on, it is not easy to follow, especially these CSS rules...
Furthermore, there is `o_website_rating_progress_table` and `o_website_rating_table_progress` :face_with_spiral_eyes: . Do you think uniformizing it to something like this is useful? or would it cause trouble somewhere else?

```xml
<div class="o_website_rating_table o_website_rating_table_has_selection">
    <div class="o_website_rating_table_row">...</div>
    <div class="o_website_rating_table_row o_website_rating_table_row_selected">
        <div class="o_website_rating_table_star_num" />
        <div class="o_website_rating_table_progress" />
        <div class="o_website_rating_table_percent" />
        <div class="o_website_rating_table_reset" />
    <div class="o_website_rating_table_row">...</div>
    ...
  </div>
</div>
```

Reply:
Thanks for your proposition which is not too much changes while improving the readability.